[17][ixe2txxwil] 로그인 시작
[17][ixe2txxwil] 로그인 종료
[17][ixe2txxwil] 로그인 검증 시작
[17][ixe2txxwil] 동의여부 체크 시작
[17][ixe2txxwil] Traceback (most recent call last):
  File "/instahelp_web/common/fun.py", line 894, in login
    if agree_check(account_id, tab_index, driver, wait):
  File "/instahelp_web/common/fun.py", line 803, in agree_check
    if agree.text == 'Instagram 앱을 사용하려면 다음 항목에 동의해야 합니다':
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 90, in text
    return self._execute(Command.GET_ELEMENT_TEXT)["value"]
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webelement.py", line 395, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.9/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.StaleElementReferenceException: Message: stale element reference: stale element not found in the current frame
  (Session info: chrome-headless-shell=126.0.6478.61); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
Stacktrace:
#0 0x561448f266aa <unknown>
#1 0x561448c090dc <unknown>
#2 0x561448c0e7e9 <unknown>
#3 0x561448c10595 <unknown>
#4 0x561448c1061c <unknown>
#5 0x561448c4fae2 <unknown>
#6 0x561448c78872 <unknown>
#7 0x561448c49d89 <unknown>
#8 0x561448c78a3e <unknown>
#9 0x561448c975c3 <unknown>
#10 0x561448c78613 <unknown>
#11 0x561448c484f7 <unknown>
#12 0x561448c48e4e <unknown>
#13 0x561448eec87b <unknown>
#14 0x561448ef0921 <unknown>
#15 0x561448ed836e <unknown>
#16 0x561448ef1482 <unknown>
#17 0x561448ebcccf <unknown>
#18 0x561448f160a8 <unknown>
#19 0x561448f16280 <unknown>
#20 0x561448f257dc <unknown>
#21 0x7fbe060a1134 <unknown>


[17][ixe2txxwil] Task[0,1,로그인 실패]
[17][ixe2txxwil] 마지막 메모리 사용량: 13.02 MB
